src/Tests/Microsoft.NET.TestFramework/RepoInfo.cs

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.NET.TestFramework
+{
+    public class RepoInfo
```

Comment:
I think Daniel deleted this file. Is this a bad merge that resurrected it?

Reply:
potentially.